nrpy/infrastructures/CarpetX/general_relativity/floor_the_lapse.py

```diff
@@ -0,0 +1,83 @@
+"""
+Generates a function to enforce a minimum on the lapse grid function.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+from typing import Union, cast
+from inspect import currentframe as cfr
+from types import FrameType as FT
+
+import nrpy.c_function as cfc
+import nrpy.grid as gri
+import nrpy.helpers.parallel_codegen as pcg
+import nrpy.infrastructures.CarpetX.simple_loop as lp
+
+standard_ET_includes = [
```

Comment:
Centralize.

Reply:
Centralized in `CarpetX_include_header.py`